app/controllers/api/notify_callbacks_controller.rb

```diff
@@ -2,30 +2,23 @@
 
 class Api::NotifyCallbacksController < Api::ApiController
   def create
-    return head :no_content unless params[:reference] || params[:id]
+    return head :no_content unless params[:id]
 
-    if params[:reference]
-      email = NominationEmail.find_by(token: params[:reference])
-      email.update!(notify_status: params[:status]) if email
-    else
-      email = PartnershipNotificationEmail.find_by(notify_id: params[:id])
-      email.update!(notify_status: params[:status], delivered_at: params[:sent_at]&.to_datetime) if email
-    end
+    log_email if failed_email?
 
-    log_email if failed_email?(email)
+    email = PartnershipNotificationEmail.find_by(notify_id: params[:id]) || NominationEmail.find_by(notify_id: params[:id])
```

Comment:
Do we also need to update `notify_id` on NominationEmails somewhere that's sending them?

Reply:
I think we can remove the reference param in the mailer too

Reply:
Good spot both